clab/clab.go

```diff
@@ -207,12 +207,14 @@ func (c *CLab) ProcessTopoPath(path string) (string, error) {
 
 	switch {
 	case path == "-" || path == "stdin":
+		log.Debugf("interpretting topo %q as stdin", path)
 		file, err = readFromStdin(c.TopoPaths.ClabTmpDir())
 		if err != nil {
 			return "", err
 		}
 	// if the path is not a local file and a URL, download the file and store it in the tmp dir
 	case !utils.FileOrDirExists(path) && utils.IsHttpURL(path, true):
+		log.Debugf("interpretting topo %q as remote URL", path)
```

Comment:
👋 @matthewCmatt 

maybe also add the check that the string does not contain a dot char in the `utils.IsHttpURL` to make sure we don't treat `abc/def.clab.yml` as a file url?